api/db-cdr/csv/achilles_analysis.csv

```diff
@@ -17,7 +17,7 @@ analysis_id,"analysis_name","stratum_1_name","stratum_2_name","stratum_3_name","
 3005,"Counts per person distribution by concept id","measurement_concept_id",NULL,NULL,NULL,NULL,"box","distribution"
 3006,"Counts per person by gender distribution","measurement_concept_id","gender_concept_id",NULL,NULL,NULL,"box","distribution"
 3007,"Counts per person by age at occurrence","measurement_concept_id","age_decile_at_occurrence",NULL,NULL,NULL,"box","distribution"
-3008,"Counts per person distrubtion by age at occurrence and gender","measurement_concept_id","age_decile_at_occurrence","gender_concept_id",NULL,NULL,"box","distribution"
+3008,"Counts per person distrubtion by age at occurrence and gender","measurement_concept_id","age_decile_at_occurrence","gender",NULL,NULL,"box","distribution"
```

Comment:
is this not a gender concept id anymore?  And does the db-cdr/generate-cdr need these changes ?